src/lib/components/ExpertsView.tsx

```diff
@@ -48,18 +45,13 @@ export const ExpertsView: React.FC<IExpertsViewProps> = (props) => {
     </div>
   ));
 
-  const renderError = (errorMsg = 'Не вдалося завантажити експертів') =>
-    loading === LoadingStatusEnum.failed ? <span>{errorMsg}</span> : null;
-
-  const renderLoading = () =>
-    loading === LoadingStatusEnum.pending ? <CircularProgress /> : null;
+  const errorMsg = 'Не вдалося завантажити експертів';
 
   return (
     <Container>
       <Typography variant="h4">Експерти</Typography>
       <Grid container direction="column" alignItems="center">
-        {renderLoading()}
-        {renderError()}
+        {loading && <LoadingInfo loading={loading} errorMsg={errorMsg} />}
       </Grid>
       <div className={classes.container}>
```

Comment:
i guess this should not be shown when `loading`

Reply:
It's strange, but I don't have these 61 & 62 lines in my code.  I've pushed again.

Reply:
no it's okay :)
i mean, not to show all experts container on line:56 when `loading`

Reply:
fixed